_src/static/sass/components/_headings.scss

```diff
@@ -0,0 +1,128 @@
+// ------------------------------
+// edX Pattern Library: Components - Headings
+
+// About: Contains base styling for headings, relying on Bourbon's mathematical scales for precision.
+
+// Scale:
+// 12, 14, 16, 18, 21, 24, 32, 43
+// ----------------------------
+
+// #GLOBALS
+// #HEADINGS
+// H1
+// H2
+// H3
+// H4
+// H5
+// H6
+// ----------------------------
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6 {
+    font-family: $headings-base-font-family;
+    font-weight: $headings-base-font-weight;
+    color: $headings-base-color;
+
+    &.emphasized,
+    &.primary {
+        font-weight: $headings-primary-font-weight;
+        color: $headings-primary-color;
+    }
+
+    &.deemphasized,
+    &.secondary {
+        font-weight: $headings-secondary-font-weight;
+        color: $headings-secondary-color;
+    }
+}
+
+.hd-5,
+.hd-6 {
+    font-weight: $headings-primary-font-weight;
+    text-transform: uppercase;
+}
+
+
+// ----------------------------
+// #HEADINGS
+// ----------------------------
+// H1
```

Comment:
NIT: these comments shouldn't have this formatting. They should match that of https://github.com/edx/ux-pattern-library/blob/master/_src%2Fstatic%2Fsass%2Futilities%2F_variables.scss#L283. So more like this:

````scss
// ----------------------------
// #HEADINGS
// ----------------------------
// Level one heading
.hd-1 {
    ...
}

// Level two heading
.hd-2 {
    ...
}

...

// Level six heading
.hd-6 {
    ...
}

````